web.rb

```diff
@@ -102,6 +102,26 @@ def authenticate!
           :my_customer_id => '72F8C533-FCD5-47A6-A45B-3956CA8C792D',
         },
       )
+      # Attach some test cards to the customer for testing convenience.
+      # See https://stripe.com/docs/payments/3d-secure#three-ds-cards
+      ['4000000000003220', '4242424242424242'].each { |number|
```

Comment:
ooc, is it possible to use the token or PaymentMethod id version of these? (see the Tokens/Payment Methods tabs)

Reply:
Ooh yep, that would be simpler!